src/Core/Entity/FormStats.php

```diff
@@ -21,6 +21,7 @@ class FormStats extends StaticEntity
     protected $total_messages_sent;
     protected $total_messages_pending;
     protected $total_by_data_source;
+
```

Comment:
I don't think this file needs to change

Reply:
oh. phpcs was complaining about spacing that's why it changed